STYLE.md

```diff
@@ -77,6 +77,9 @@ following abbreviations we assume readers will already know.
 #### Forbidden abbreviations
```

Comment:
Perhaps this should be "Forbidden terms" or similar, since these aren't really abbreviations.